packages/server/tests/acceptance/htsPrecompile.spec.ts

```diff
@@ -604,4 +604,162 @@ describe('HTS Precompile Acceptance Tests', async function () {
+    });
+
+    it('should be able to transfer non-fungible tokens', async function() {
+      // Mint an NFT
+      const tx = await baseHTSContract.mintTokenPublic(NftHTSTokenContractAddress, 0, ['0x03', '0x04'], { gasLimit: 1_000_000 });
+      expect((await tx.wait()).events.filter(e => e.event === 'ResponseCode')[0].args.responseCode).to.be.equal(TX_SUCCESS_CODE);
+      const { serialNumbers } = (await tx.wait()).events.filter(e => e.event === 'MintedToken')[0].args;
+      NftSerialNumber = serialNumbers[0];
+      NftSerialNumber2 = serialNumbers[1];
+
+      await setKyc(NftHTSTokenContractAddress);
+        
+      // setup the transfer
+      const tokenTransferList = [{
+        token: `${NftHTSTokenContractAddress}`,
+        transfers: [],
+        nftTransfers: [{
+          senderAccountID: `${accounts[0].wallet.address}`,
+          receiverAccountID: `${accounts[1].wallet.address}`,
+          serialNumber: NftSerialNumber.toNumber(),
+        },
+        {
+          senderAccountID: `${accounts[0].wallet.address}`,
+          receiverAccountID: `${accounts[2].wallet.address}`,
+          serialNumber: NftSerialNumber2.toNumber(),
+        }],
+      }];
+      const txXfer = await baseHTSContract.cryptoTransferPublic(tokenTransferList);
+      expect((await txXfer.wait()).events.filter(e => e.event === 'ResponseCode')[0].args.responseCode).to.equal(TX_SUCCESS_CODE);
+    });
+
+    it('should be able to transfer both fungible and non-fungible tokens in single cryptoTransfer', async function() {
+      // Mint an NFT
+      const tx = await baseHTSContract.mintTokenPublic(NftHTSTokenContractAddress, 0, ['0x05'], { gasLimit: 1_000_000 });
+      expect((await tx.wait()).events.filter(e => e.event === 'ResponseCode')[0].args.responseCode).to.be.equal(TX_SUCCESS_CODE);
+      const { serialNumbers } = (await tx.wait()).events.filter(e => e.event === 'MintedToken')[0].args;
+      NftSerialNumber = serialNumbers[0];
+        
+      // setup the transfer
+      const tokenTransferList = [{
+        token: `${NftHTSTokenContractAddress}`,
+        transfers: [],
+        nftTransfers: [{
+          senderAccountID: `${accounts[0].wallet.address}`,
+          receiverAccountID: `${accounts[1].wallet.address}`,
+          serialNumber: NftSerialNumber.toNumber(),
+        }],
+      },
+      {
+        token: `${HTSTokenContractAddress}`,
+        transfers: [
+          {
+            accountID: `${accounts[1].wallet.address}`,
+            amount: 10,
+          },
+          {
+            accountID: `${accounts[0].wallet.address}`,
+            amount: -10,
+          },
+        ],
+        nftTransfers: [],
+      }];
+      const txXfer = await baseHTSContract.cryptoTransferPublic(tokenTransferList);
+      expect((await txXfer.wait()).events.filter(e => e.event === 'ResponseCode')[0].args.responseCode).to.equal(TX_SUCCESS_CODE);
+    });
+
+    xit('should be able to swap non-fungible tokens', async function() {        
```

Comment:
Let it run and check for failure like you do in the other.
Then when it's fixed we'll know to just update
```suggestion
    it('should be able to swap non-fungible tokens', async function() {        
```